src/commands/config/config-template.json

```diff
@@ -0,0 +1,5 @@
+{
+  "network": "ropsten",
+  "wallet": "<Wallet string>",
```

Comment:
no need this default value

Reply:
ok removed this file as its empty and useless